### PR TITLE
added signin styles

### PR DESCRIPTION
**Pull request by grantis**
Added Material UI core for signin Styles, 
Was required to update the Router in App to render the component properly.

### CODE REVIEW

**Comment by grantis** [comment]
I changed the browserRouter to HashRouter to get this working and rendering and I update the app.js file to correctly render the signin component.

File: client/components/App.jsx [hunk 1]
```diff
@@ -9,9 +9,9 @@ const App = () => {
   return (
     <React.Fragment>
       <h1>React development has begun!</h1>
-      <Route SignIn path="/signin" component={SignIn} />
-      <Route Dashboard path="/" component={Dashboard} />
-      <Route path="/register" component={RegistrationForm} />
+      <Route exact path="/signin" component={SignIn} />
```
Comment (emilyparkes) [inline client/components/App.jsx:12]: this should be using a <Switch> statement from react-router-dom wrapped around all routes, not exact path.
Also is this part of your ticket?

File: client/components/SignIn.jsx [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
-        <input type="text" name="username" />
-        <br />
-        <br />
-        <span>Password </span>
-        <input type="text" name="password" />
-        <br />
-        <br />
-        <input type="submit" value="Submit" />
-      </form>
+      <h1>SIGN IN COMPONENT</h1>
```
Comment (emilyparkes) [inline client/components/SignIn.jsx]: Remove please, we do not need this in our dev branch as the user will not see this.

File: client/components/SignIn.jsx [hunk 3]
```diff
@@ -1,20 +1,23 @@
...
+        varient="outlined"
+        label="firstname" />
+      <TextField
+        varient="outlined"
+        label="lastname" />
+      <TextField
+        varient="outlined"
+        label="password" />
+      <Button variant="contained" color="primary">
+        Login
```
Comment (emilyparkes) [inline client/components/SignIn.jsx:18]: sign in or log in? Please keep naming the same.

File: client/index.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
-import {BrowserRouter as Router} from 'react-router-dom'
+import { HashRouter as Router } from 'react-router-dom'
```
Comment (emilyparkes) [inline client/index.js]: nope. We are using browser router. Please don't change this.

**Review by emilyparkes — CHANGES_REQUESTED** [review]

**Comment by emilyparkes** [comment]
> I changed the browserRouter to HashRouter to get this working and rendering and I update the app.js file to correctly render the signin component.

This is not how you should fix Browser Router - by replacing it with something else, you should fix it correctly. Luckily it has been done and is on the dev branch now.

**Pull request closed without merging**